rust/metaplex/program/src/processor/empty_payment_account.rs

```diff
@@ -85,11 +85,23 @@ fn calculate_owed_amount(
     winning_config_item_index: &Option<u8>,
     creator_index: &Option<u8>,
 ) -> Result<u64, ProgramError> {
-    let primary_sale_happened = auction_manager.get_primary_sale_happened(
-        metadata,
-        *winning_config_index,
-        *winning_config_item_index,
-    )?;
+    msg!(
+        "safety_deposit_config_info: {:?}",
+        safety_deposit_config_info
+    );
```

Comment:
This prints a lot of text and most of it is not useful, we should probably remove this from the contract.